src/NLog/Targets/TraceTarget.cs

```diff
@@ -62,6 +63,12 @@ namespace NLog.Targets
     public sealed class TraceTarget : TargetWithLayout
     {
         /// <summary>
+        /// Always use <see cref="Trace.WriteLine(string)"/> independent of <see cref="LogLevel"/>
+        /// </summary>
+        [DefaultValue(false)]
+        public bool WriteLine { get; set; }
```

Comment:
mmmm, sounds like IgnoreLogLevel=error.

Don't want to be picky, do you have another idea?

What about EnablePrefix, DontSerializeLogLevel, SkipLogLevelPrefix, RawLayoutOnly,...